From b7d949f65d9ead9feb127a14bae9ab78831e99df Mon Sep 17 00:00:00 2001
From: Wang Quanyang <quanyang.wang@windriver.com>
Date: Fri, 12 Jul 2019 05:17:18 -0400
Subject: [PATCH] gpu: mail-400: remove type argument from access_ok

Because of upstream commit 96d4f267e40f ("Remove 'type' argument from access_ok() function"),
remove type from access_ok.

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 linux/mali_ukk_mem.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/linux/mali_ukk_mem.c b/linux/mali_ukk_mem.c
index b0b989c..07186b3 100644
--- a/linux/mali_ukk_mem.c
+++ b/linux/mali_ukk_mem.c
@@ -207,8 +207,8 @@ int mem_write_safe_wrapper(struct mali_session_data *session_data, _mali_uk_mem_
 	kargs.ctx = (uintptr_t)session_data;
 
 	/* Check if we can access the buffers */
-	if (!access_ok(VERIFY_WRITE, kargs.dest, kargs.size)
-	    || !access_ok(VERIFY_READ, kargs.src, kargs.size)) {
+	if (!access_ok(kargs.dest, kargs.size)
+	    || !access_ok(kargs.src, kargs.size)) {
 		return -EINVAL;
 	}
 
@@ -266,7 +266,7 @@ int mem_dump_mmu_page_table_wrapper(struct mali_session_data *session_data, _mal
 		goto err_exit;
 
 	user_buffer = (void __user *)(uintptr_t)kargs.buffer;
-	if (!access_ok(VERIFY_WRITE, user_buffer, kargs.size))
+	if (!access_ok(user_buffer, kargs.size))
 		goto err_exit;
 
 	/* allocate temporary buffer (kernel side) to store mmu page table info */
-- 
2.22.0

